### DISCUSSION TITLE
Api documentation tab not visible for rest api's

### DISCUSSION THREAD

**Discussion by rohit-raje-786** [hoppscotch/hoppscotch#2858] [Q&A] [answered]
I  want to publish an api,but I'm not able to find the documentation tab for the rest API collections.
![image](https://user-images.githubusercontent.com/61861303/202840215-10f55435-9eb3-4e8b-8a6a-ef7d3df6655f.png)

How do I create API documentation for rest api's?

**Comment by liyasthomas** [comment, marked as answer]
We’re revamping API documentation flow. Temporarily disabled API documentation page visibility for the refactor.

Expect a brand new API documentation experience soon. Stay tuned for product updates on GitHub.

**Reply by pycharmer1221** [reply]
is there any updates on API documentation  ?

also let me know if there is any solution to unable it in self-hosted environment

**Comment by rajithavk** [comment]
I'm also waiting for this feature. What's the status as of now if you'd like to share?
